src/toaster/src/ToastManager.js

```diff
@@ -89,6 +81,12 @@ export default class ToastManager extends React.PureComponent {
     return instance
   }
 
+  checkPosition = ({ position }) => {
+    if (position) {
```

Comment:
Notably this will mean that if you set a position once, you'll have to reset the position every time you want it to be something else.

I think perhaps a nicer way to introduce this would be to only apply the position to the one instance that has the setting (and not persist the position in state). Thoughts?